cvat/apps/engine/models.py

```diff
@@ -320,6 +345,12 @@ class Meta:
         default_permissions = ()
         unique_together = ('task', 'name')
 
+
+class TrainingProjectLabel(models.Model):
+    cvat_label = models.ForeignKey(Label, on_delete=models.CASCADE, related_name='training_project_label')
+    training_label_id = models.CharField(max_length=64)
```

Comment:
Why not int?

Reply:
label_id_example = 5e958dbc563d85f77129aab8